cmc-domain-model/src/main/java/uk/gov/hmcts/cmc/claimstore/constraints/MobilePhoneNumberConstraintValidator.java

```diff
@@ -16,10 +16,14 @@ public boolean isValid(final String value, final ConstraintValidatorContext cxt)
             return true;
         }
 
-        String phone = value.replaceAll("[^0-9]", "")
+        System.out.println(value);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.reform.hmcts.net/coding_rules#rule_key=squid%3AS106)